Modules/TPC/src/Clusters.cxx

```diff
@@ -199,17 +217,41 @@ void Clusters::monitorData(o2::framework::ProcessingContext& ctx)
 
   mQCClusters.analyse();
 
-  int histIter = 0;
-  for (auto& calPadWrapper : mWrapperVector) {
-    auto& histA = *mHistoVector[histIter].get();
-    auto& histC = *mHistoVector[histIter + 1].get();
-    const auto& calPad = calPadWrapper.getObj();
+  std::vector<TCanvas*> vecPtrNClusters;
```

Comment:
See comment in 
`void Calibration::update(Trigger t, framework::ServiceRegistry&)`